dashboard/views.py

```diff
@@ -6,8 +6,12 @@
 @login_required
 def dashboard(request):
     current_user = request.user
-    available_plans = []
+    topic_to_plans = {}
+
     for plan in LessonPlan.objects.all():
         if current_user in plan.assigned_users.all():
-            available_plans.append({"name": plan.title, "id" : plan.id, "topics" : plan.topics.all()})
+            for topic in plan.topics.all() :
+                topic_to_plans.setdefault(topic, []).append({"id" : plan.id, "title": plan.title, "aim":plan.aim})
+                # topic_to_plans.setdefault(topic, []).append({"id" : plan.id, "title": plan.title, "aim":plan.aim, "topics" : plan.topics.all()})
+        
     return render(request, "dashboard/index.html", {"available_plans" : available_plans})
```

Comment:
It should be `{"topic_to_plans" : topic_to_plans}` instead of `"available_plans" : available_plans}`.

Reply:
Done 👍🏻